lib/utils/env.ts

```diff
@@ -1,10 +1,10 @@
 import { isValidCollectionCodename } from "../types/perCollection";
 
-const { KONTENT_COLLECTION_CODENAME } = process.env;
+const { NEXT_PUBLIC_KONTENT_COLLECTION_CODENAME } = process.env;
 
-if (!isValidCollectionCodename(KONTENT_COLLECTION_CODENAME)) {
-  throw new Error(`Invalid collection codename "${KONTENT_COLLECTION_CODENAME}".`);
+if (!isValidCollectionCodename(NEXT_PUBLIC_KONTENT_COLLECTION_CODENAME)) {
+  throw new Error(`Invalid collection codename "${NEXT_PUBLIC_KONTENT_COLLECTION_CODENAME}".`);
 }
 
 /** Use only on server - for client use `useSiteCodename` hook */
```

Comment:
Since the env variable is now public this comment and the `useSiteCodename` hook doesn't make sense anymore. Can you please remove the hook and replace its usages with this constant? It will simplify the app.

Reply:
I decided to remove filtering based on current siteCodename 46d0e73 and it seems to be working
* for the current showcase it is fine
* i haven't found error message for links bein incorrectly reolsved

Reply:
That's a bit heavier on the local run configuration (when I change collection in `.env` I also have to change the URLs variable, whereas in the previous version, I didn't have to). But I can live with that. :)

Reply:
This is because of deployment to Vercel - this would lead to inconsistent local links.